tests/testsuite.src/syn_copy.at

```diff
@@ -1072,3 +1072,36 @@ AT_DATA([prog.cob], [
 AT_CHECK([$COMPILE_ONLY prog.cob], [0], [], [])
 
 AT_CLEANUP
+
+
+AT_SETUP([Option --copy COPYBOOK])
+AT_KEYWORDS([argument])
+
+AT_DATA([copybook.cpy], [
+       REPLACE ==BEGIN PROGRAM== BY ==IDENTIFICATION DIVISION.
+       PROGRAM-ID.==
+       ==output== BY =="Hello world"==.
+])
+AT_DATA([prog.cob], [
+       BEGIN PROGRAM    prog.
+       PROCEDURE        DIVISION.
+           DISPLAY output
+           STOP RUN.
+])
+
+AT_CHECK([$COMPILE_ONLY prog.cob], [1], [],
+[prog.cob:2: error: PROGRAM-ID header missing
+prog.cob:2: error: ENVIRONMENT DIVISION header missing
+prog.cob:2: error: CONFIGURATION SECTION header missing
+prog.cob:2: error: SPECIAL-NAMES header missing
+prog.cob:2: error: invalid system-name 'BEGIN'
+prog.cob:2: error: syntax error, unexpected PROGRAM, expecting CRT or Identifier
+prog.cob:2: error: invalid system-name 'prog'
+prog.cob:2: error: syntax error, unexpected ., expecting CRT or Identifier
+prog.cob:3: error: syntax error, unexpected PROCEDURE
+prog.cob:4: error: PROCEDURE DIVISION header missing
+])
+
+AT_CHECK([$COMPILE_ONLY --copy copybook prog.cob])
+
```

Comment:
Please add another file and `AT_CHECK`  below (to the same test case) where the command-line-included copybook has an error; just to ensure (and showcase) that we have an reasonable output then.